### DISCUSSION TITLE
[Svelte Template] How to Only Keep the ``frontend/dist`` and Ignores all Files in It?

### DISCUSSION THREAD

**Discussion by KiddoV** [wailsapp/wails#2088] [Q&A]
I am using ``Svelte Template``, and I would like to git-ignores all the files in the ``frontend/dist``, since everything will be automatically generate.
I have tried to put a ``.gitkeep`` in the ``dist`` folder, but whenever I re-run, the app will erase everything in that folder including the ``.gitkeep`` file.
Is there anyway to accomplish this? Thanks!

**Comment by leaanthony** [comment]
1. Remove from `.gitignore`
2. Add `.gitkeep`
3. Commit
4. Add back to `.gitignore`
5. Commit

**Reply by KiddoV** [reply]
Step 1. Remove what from ``.gitignore`` exactly?
